src/Pages/Goals.cshtml.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
```

Comment:
Using quite a few unnecessary dependancies here

Reply:
Did not pay attention to this file tbh, since Goals only have design and no functionality, **yet**, but still good notice 🙂